### ISSUE TITLE
api/v1/content/overview/namespace/default/?poll=5 - 403 forbidden

### ISSUE THREAD

**Issue by nodesocket** [vmware/octant#188]
Just shows a loading progress bar across the top.

Running Kubernetes 1.13 and confirm I am able to run `kubectl cluster-info` and `kubectl get pods`. However when I bring up the octant web interface, it just shows the loading progress bar across the top in Chrome 76 on MacOS.

The only request in Chrome Developer tools I see failing is:

```
http://127.0.0.1:8900/api/v1/content/overview/namespace/default/?poll=5
Request Method: GET
Status Code: 403 Forbidden
```

**Comment by wwitzel3** [comment]
Thanks for reporting this @nodesocket 

Can you provide some extra detail, this will help me with reproducing:
  * provide the command you are using launch Octant
  * which version of Octant are you running (0.5.1, master, etc..)
  * Can you add the `-v` flag when you launch Octant and provide the terminal output in a gist.

**Comment by nodesocket** [comment]
Hi @wwitzel3.

```
ubuntu@ip-192-168-3-161:~$ octant version
Version:  0.5.1
Git commit:  e0397d2596a7575abf4ca9aeacafd92912d484e0
Built:  2019-08-06T19:40:35Z
```

Running with `-v`:

https://gist.github.com/nodesocket/bac50ccb67e70b0fc1cf5f0ad249338e

Screenshot of the UI with the 403 error:

<img width="1151" alt="Screen Shot 2019-08-20 at 12 49 20 PM" src="https://user-images.githubusercontent.com/523312/63371094-0a0a8000-c349-11e9-942a-082d1236bbe0.png">

**Comment by wwitzel3** [comment]
Thanks, this was helpful. We fixed #152 and added #194 which will be apart of the 0.6 release, which should be out later today.

Then you'll be able to set the `OCTANT_LISTENER_ADDR` and add any hosts  to the `OCTANT_ACCEPTED_HOSTS` list.

**Comment by wwitzel3** [comment]
See: https://github.com/vmware/octant/blob/master/docs/getting-started.md#environment-variables

For details on the available environment variables.

**Comment by wwitzel3** [comment]
This is fixed in master and will be in our 0.6 release.

**Comment by nodesocket** [comment]
@wwitzel3 so what should I set `OCTANT_ACCEPTED_HOSTS` to ? I enforce access control permissions at the firewall level.

**Comment by wwitzel3** [comment]
I think in this case, you'd set it `192.168.3.161`?

**Comment by wwitzel3** [comment]
Essentially you want it set to what ever the value of `request.Host` will be, so if you're doing any traffic re-writing or routing, you'll need to take that in to account.

**Comment by wwitzel3** [comment]
I opened a new issue related to this, if you start Octant listening on 0.0.0.0 is should allow all hosts. See #199

**Comment by GuessWhoSamFoo** [comment]
@nodesocket Not sure if it helps, I wrote some notes https://github.com/vmware/octant/issues/199#issuecomment-523563021

**Comment by zwunan** [comment]
I also met this problem, how to solve it

**Comment by wwitzel3** [comment]
@zwunan we recently merged some work around this (see #200). Have you tried explicitly setting `OCTANT_ACCEPTED_HOSTS` to address of the requesting client?
